src/include/ceph_fs.h

```diff
@@ -301,6 +301,7 @@ enum {
 	CEPH_SESSION_RENEWCAPS,
 	CEPH_SESSION_STALE,
 	CEPH_SESSION_RECALL_STATE,
+	CEPH_SESSION_RECALL_STATEMSG_ACK,
```

Comment:
this needs to go at the end of the enum otherwise the binary compatibility of the state messages is compromised